src/Console/Command/ReadmeCommand.php

```diff
@@ -99,14 +99,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
 Globally (Composer)
 ~~~~~~~~~~~~~~~~~~~
 
-To install PHP CS Fixer, install Composer and issue the following command:
+To install PHP-CS-Fixer, `install Composer <https://getcomposer.org/download/>`_ and issue the following command:
```

Comment:
name of the tool is `PHP CS Fixer`
name of bin file or composer package is `php-cs-fixer`.

`PHP-CS-Fixer` is some weird combination of those.